docs/source/compression/pruning_transformer.rst

```diff
@@ -0,0 +1,54 @@
+Pruning Transformer with NNI
+============================
+
+Workable Pruning Process
+------------------------
+
+Here we show an effective transformer pruning process that NNI team has tried, and users can use NNI to discover better processes.
+The full code can be found :githublink:`here <examples/model_compress/pruning/pruning_bert_glue.py>`.
+
+During the process of pruning transformer, we gained some of the following experience:
+
+* We using :ref:`movement-pruner` in step 2 and :ref:`taylor-fo-weight-pruner` in step 4. :ref:`movement-pruner` has good performance on attention layers,
+  and :ref:`taylor-fo-weight-pruner` method has good performance on FFN layers. These two pruners are all some kinds of gradient-based pruning algorithm,
+  we also try weight-based pruning algorithms like :ref:`l1-norm-pruner`, but it doesn't seem to work well in this scenario.
+* Distillation is a good way to recover model precision. In terms of results, usually 1~2% improvement in accuracy can be achieved when we prune bert on mnli task.
+* It is necessary to gradually increase the sparsity rather than reaching a very high sparsity all at once.
+
+Result
+------
+
+.. list-table:: Prune Bert-base-uncased on MNLI
+    :header-rows: 1
+    :widths: auto
+
+    * - Attention Pruning Method
+      - FFN Pruning Method
+      - Total Sparsity
+      - Accuracy
+      - Acc. Drop
```

Comment:
how about add a column for speedup ratio?

Reply:
good suggestion, I will add them